core-backend/common/src/lib.rs

```diff
@@ -27,6 +27,9 @@ pub mod error_processor;
 mod utils;
 pub use utils::calc_stack_end;
 
+mod sys_calls;
+pub use sys_calls::SysCallNames;
```

Comment:
for me `syscalls` established expression 
```suggestion
pub use syscalls::Sysсalls;
```